src/ClassDiscovery.php

```diff
@@ -54,8 +58,10 @@ protected static function findOneByType($type)
         foreach (self::$strategies as $strategy) {
             try {
                 $candidates = call_user_func($strategy.'::getCandidates', $type);
-            } catch (StrategyUnavailableException $e) {
-                $exceptions[] = $e;
+	    } catch (StrategyUnavailableException $e) {
```

Comment:
Please increase indentation.

Reply:
Ugh that's using tabs. New machine, editor not configured properly yet XD